src/persist/src/indexed/trace.rs

```diff
@@ -110,7 +162,11 @@ impl<K: Data, V: Data> BlobTrace<K, V> {
         let desc = batch.desc.clone();
         let key = self.new_blob_key();
         blob.set_trace_batch(key.clone(), batch)?;
-        self.batches.push((desc, key));
+        self.batches.push(BlobTraceBatchMeta {
+            key,
+            desc,
+            level: 1,
```

Comment:
Toss in a comment to the effect of "As mentioned in the big comment on BlobTrace, a batch's initial level is set to 1."